src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -215,6 +215,7 @@ elseif(CLR_CMAKE_TARGET_ANDROID AND NOT CROSS_ROOTFS)
         add_subdirectory(System.Security.Cryptography.Native)
     endif()
 else()
+    add_subdirectory(System.Globalization.Native)
```

Comment:
Do we also need to add it to the nuget package, or is that going to happen automatically?

Reply:
I haven't yet looked at whether we include any of the .a files - it does work for the part that builds out CORE_ROOT and we can start running CI with this.

But none of the threading smoke tests work so I'm not going to enable it yet.

Reply:
> I haven't yet looked at whether we include any of the .a files

We were not including them a week ago. I have looked at it some, but I was not able to figure out the right msbuild spell to get them included. If you can take a look at it, it would be great.

> But none of the threading smoke tests work so I'm not going to enable it yet.

Sorry, it is likely a regression that I have introduced in the CoreLib.Native port from CoreRT repo... .